test/integration/isolated/suite_test.go

```diff
@@ -306,7 +306,8 @@ func featureSetup(opts ...featureSetupOpt) func(ctx context.Context, t *testing.
 			fmt.Sprintf("--admission-webhook-listen=0.0.0.0:%d", testutils.AdmissionWebhookListenPort),
 			"--anonymous-reports=false",
 			fmt.Sprintf("--feature-gates=%s", featureGates),
-			fmt.Sprintf("--election-namespace=%s", kongAddon.Namespace()),
+			// Use fixed election namespace because RBAC roles for leader election are in the namespace
```

Comment:
Can we make this comment clearer. Is this trying to comment the fact that KIC election resources are to be created in its namespace which is always: `consts.ControllerNamespace`?

Reply:
The role is always `consts.ControllerNamespace` if we do not change the code for deploying the controller, but it is easier to change the namespace of resources for leader election.